HW_6/packages/turns/u_t.py

```diff
@@ -0,0 +1,24 @@
+# user turns
+def user_turn(board, checklist):
+    turn_check = True
+    while turn_check:
+        u_turn = int(input("enter the number of nonmaked box U want to mark: "))
+        if u_turn not in checklist:
+            print("incorrect box number")
+            continue
+        else:
+            turn_check = False
+            checklist[u_turn - 1] = "O"
+            if (u_turn - 1) // 3 == 0:
+                big_l = 0
+            elif (u_turn - 1 )// 3 == 1:
+                big_l = 1
+            elif (u_turn - 1) // 3 == 2:
+                big_l = 2
+            if u_turn % 3 == 1:
+                mini_l = 0
+            elif u_turn % 3 == 2:
+                mini_l = 1
+            elif u_turn % 3 == 0:
+                mini_l = 2
```

Comment:
Can be improved just like that:

```suggestion
u_turn = u_turn - 1
row = u_turn // 3 
col = u_turn % 3
sign = board[row][col]
ok = sign not in ['O','X'] 
if not ok:	# it's occupied - to the input again
    print("repeat your input!")
    continue
board[row][col] = 'O' 
```